### PR TITLE
docs: add DMP Week 01 progress post Nirav Sharma

### PR DESCRIPTION
**Pull request by 021nirav-blip**
This is the week01 update for the Refactor Temperament project.

The short version: PR 1 is merged, PR 2 is in progress, and everything
still works.

PR #7561 landed on June 18. The core pitch math now handles any EDO
instead of assuming 12. I added 23 unit tests, and all 5561 existing
tests are still green. 12-EDO behavior is unchanged.

Walter's review caught a couple of things - the 1200-cent octave
constant needed a comment, and I'd put tests in the wrong folder.
Both fixed.

PR 2 is about halfway there. Six of the ten remaining functions in
musicutils.js are done. The pattern is straightforward: find hardcoded
`12`, swap it for `currentEDO`. The tricky one still to figure out is
getSolfege – we need to decide how to name pitches beyond 12.

### CODE REVIEW

File: src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Devin Ulibarri](https://github.com/pikurasa)  
+**Reporting Period:** 2026-06-17 - 2026-06-23
+
+---
+
+## What I worked on this week
+
+### PR 1 merged on June 18 🎉
+
+The first PR was merged with the title **"feat: make pitch/frequency conversion temperament-aware"**. It replaced hardcoded 12-EDO assumptions in the core pitch math functions and included important review feedback from Walter.
```
Comment (pikurasa) [inline src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md]: You should link "first PR" to the PR you are referencing, so that a reader will know what you are referring to.

File: src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+- Added `getCurrentEDO()` helper — returns `TEMPERAMENT[temperament]?.pitchNumber ?? 12`
+- Refactored `pitchToFrequency()` — now uses `Math.pow(2, 1/currentEDO)`
+- Refactored `pitchToNumber()` — now uses `octave * currentEDO`
+- Refactored `frequencyToPitch()` — now uses dynamic `centsPerStep = 1200/currentEDO`
+- Added 23 unit tests covering 12-EDO, 19-EDO, default behavior, and edge cases
+
+Walter reviewed the PR and asked about the 1200-cent octave assumption, which is correct and remains valid for any EDO. He also asked to move the new tests into `js/utils/__tests__/`, which I completed.
```
Comment (pikurasa) [inline src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md]: It would be helpful if you added a sentence explaining why the 1200-cent octave assumption remains correct and valid.

File: src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+## Challenges or open questions
+
+### `getSolfege` — names for pitches beyond 12
+
+The `CHROMATIC_SOLFEGE` array currently has only 12 entries. For EDO > 12, I need to decide between:
+
+- keeping a 12-name approximation, or
+- extending solfege to numbered names like `Do1`, `Do2`, `Do3`
+
+This is a design question I'll confirm with my mentors.
```
Comment (pikurasa) [inline src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md]: Not something you need to change in this blogpost, but, as we discussed in the meeting yesterday, I recommend you look at how the Temperament Widget works, how it exports with default names but a user may change those names later.

File: src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- **Finish PR 2** — remaining functions `numberToPitch` fallback, `_getStepSize`, and `getSolfege` by June 28
+- **Open PR 2** for review once the code is ready
+- **Start PR 3** on June 29 — fix block-level temperament issues in `GetCurrentInterval` lines 157 and 169
+- **Continue refining scale builder design** under Goal 4
+
+---
+
+## Reflection
+
+This week was a major milestone because PR 1 is now merged into the codebase. That means the core conversion math is temperament-aware, and the work is no longer purely experimental.
```
Comment (pikurasa) [inline src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md]: Again, please link "PR 1" to something.

File: src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+PR 2 is more methodical and detailed, but the pattern is clear. I’m focusing on consistency so the remaining refactors stay stable and backward compatible.
```
Comment (pikurasa) [inline src/constants/MarkdownFiles/posts/2026-06-23-dmp-26-niravsharma-week01.md]: Please link "PR 2" to something.

**Review by pikurasa — CHANGES_REQUESTED** [review]
